src/app/search/map-friendly-values.pipe.ts

```diff
@@ -70,8 +70,8 @@ export class MapFriendlyValuesPipe implements PipeTransform {
     [
       'openData',
       new Map([
-        ['1', 'has public test data'],
-        ['0', 'no public test data'],
+        ['1', "doesn't need more data"],
+        ['0', 'needs more data'],
```

Comment:
Not really a fan of the contraction in the facet. 
How about "requires non-open data", "open data"

Reply:
I think it looks nice/ is clear:
![image](https://user-images.githubusercontent.com/97123241/223163358-f354a54c-6057-481a-9d81-77d06badcf07.png)

Reply:
Instead of "non-open data", maybe "restricted data" - you used it in a tooltip I think